Create-Login-and-Register-System-with-C#-Winforms/index.md

```diff
@@ -0,0 +1,430 @@
+                da2.Fill(tempds);
+
+                //======================================================================================
+
+
+            }
+            catch (Exception err) { MessageBox.Show(err.Message); }
+        }
+
+        // function to bring selected results based on column name and row index
+        public string Results(int ROW, string COLUMN_NAME)
+        {
+            try
+            {
+                return dt.Rows[ROW][COLUMN_NAME].ToString();
+            }
+            catch (Exception err)
+            {
+                MessageBox.Show(err.Message);
+                return "";
+
+            }
+        }
+
+        // function to bring selected results based on column index and row index
+        public string Results(int ROW, int COLUMN_NAME)
+        {
+            try
+            {
+                return dt.Rows[ROW][COLUMN_NAME].ToString();
+            }
+            catch (Exception err)
+            {
+                MessageBox.Show(err.Message);
+                return dt.Rows[ROW][COLUMN_NAME].ToString();
+
+            }
+        }
+
+        // Execute select statement
+        public void ExecuteSelect(string Sql_command)
+        {
+            RecordSource = Sql_command;
+            ConnectionType = Table;
+
+            dt = new DataTable(ConnectionType);
+            try
+            {
+                string command = RecordSource.ToUpper();
+                MySqlDataAdapter da = new MySqlDataAdapter(RecordSource, connection);
+                ds = new DataSet();
+                da.Fill(ds, ConnectionType);
+                da.Fill(dt);
+                tempdata = new DataGridView();
+            }
+            catch (Exception err)
+            {
+                MessageBox.Show(err.Message);
+            }
+
+
+        }
+
+        // count Number of rows after selecy
+        public int Count()
+        {
+            return dt.Rows.Count;
+        }
+    }
+}
+
+```
+
+- Now that we have Config.cs we can execute any MySQL statement 
+
+### Step IV: Create Register Windows form.
+
+From Microsoft Visual Studio create a windows form by clicking on PROJECT -> Add Windows Form, give the form a name `Register`  then click Add.
+
+We have two Windows form classes that is Form1.cs and Register.cs.
+
+### Step V: Design Login and Register interface.
+
+### Login form
+
+- Click on Form1.cs in Solution Explorer and on the form that displays add 3 buttons, 2 textboxes, and 2 labels.
+
+The first button will be the register button will start our Register form, the second one will be the Login button which will query from the input Entered and the third button will close the application.
+
+The first textbox will allow input of username for login while the second textbox will allow input of a password.
+
+The labels will indicate the functionality of the two textboxes.
+
+![LOGIN INTERFACE](/Create-Login-and-Register-System-with-C#-Winforms/login-interface.jpg)
+
```

Comment:
- This screenshot (/Create-Login-and-Register-System-with-C#-Winforms/login-interface.jpg), be a little creative, you can't have register and login buttons that way. Users opt to login because they already have an account.  
- You can have something like, "you already have an account? Login", and the same applies to the registration form.

Reply:
solved